lib/src/components/NxGlobalSidebar/__tests__/NxGlobalSidebar.test.tsx

```diff
@@ -5,98 +5,101 @@
  * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
  */
 import React from 'react';
-import { ShallowWrapper } from 'enzyme';
-import 'jest-enzyme';
-import { faCrow, faBiohazard } from '@fortawesome/free-solid-svg-icons';
-
-import * as enzymeUtils from '../../../__testutils__/enzymeUtils';
 import NxGlobalSidebar, { Props } from '../NxGlobalSidebar';
-import NxButton from '../../NxButton/NxButton';
-import NxFontAwesomeIcon from '../../NxFontAwesomeIcon/NxFontAwesomeIcon';
+import { faBiohazard, faCrow } from '@fortawesome/free-solid-svg-icons';
+import { rtlRender, runTimers, userEvent } from '../../../__testutils__/rtlUtils';
+import { screen } from '@testing-library/react';
 
 describe('NxGlobalSidebar', function() {
-  let minimalProps: Props,
-      toggleOpenSpy: () => void,
-      getShallowComponent: (optionalProps?: Partial<Props>) => ShallowWrapper;
-
-  beforeEach(function() {
-    toggleOpenSpy = jest.fn();
-    minimalProps = {
-      isOpen: false,
-      toggleOpenIcon: faCrow,
-      toggleCloseIcon: faBiohazard,
-      onToggleClick: toggleOpenSpy,
-      logoImg: 'logoImg',
-      logoAltText: 'alt text',
-      logoLink: '/ref'
-    };
-    getShallowComponent = enzymeUtils.getShallowComponent<Props>(NxGlobalSidebar, minimalProps);
-  });
+  const minimalProps: Props = {
+    isOpen: false,
+    toggleOpenIcon: faCrow,
+    toggleCloseIcon: faBiohazard,
+    onToggleClick: jest.fn,
+    logoImg: 'logoImg',
+    logoAltText: 'alt text',
+    logoLink: '/ref'
+  };
 
-  it('renders an aside with nx-global-sidebar class', function() {
-    expect(getShallowComponent()).toMatchSelector('aside.nx-global-sidebar');
-  });
+  const quickRender = rtlRender<Props>(NxGlobalSidebar, minimalProps);
 
-  it('adds classes based on isOpen state', function() {
-    expect(getShallowComponent({ isOpen: true })).toMatchSelector('.nx-global-sidebar.open');
-    expect(getShallowComponent({ isOpen: false })).toMatchSelector('.nx-global-sidebar.closed');
+  it('renders a top-level element with role="complementary"', function() {
+    expect(quickRender().getByRole('complementary')).toBeInTheDocument();
```

Comment:
Tests about the "top-level" element should check that `view.container.firstElementChild` is the element you're looking at

Reply:
> Tests about the "top-level" element should check that `view.container.firstElementChild` is the element you're looking at

Added here: bb0dd225751fb3de34eea9bf2d344495195a83f4